run_dir/design/base_b5.html

```diff
@@ -0,0 +1,135 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+
+<meta charset="utf-8">
+<title>Genomics Status</title>
+
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+<meta name="description" content="Genomics Status. Keeping track of samples in SciLifeLab">
+<meta name="author" content="Phil Ewels, Denis Moreno, Kate Stepanova,Guillermo Carrasco, Remi-André Olsen, many others.">
+
+<!-- Javascript -->
+<script src="/static/js/jquery-3.5.1.min.js"></script>
+<script src="/static/js/moment.min.js"></script>
+{% if gs_globals['font_awesome_url'] %}
+<script src={{ gs_globals['font_awesome_url'] }} crossorigin="anonymous"></script>
+{% end %}
+
+<!-- Le styles -->
+<link rel="stylesheet" href="/static/css/bootstrap-5-alpha.css" rel="stylesheet">
+<link rel="stylesheet" href="/static/css/status_b5.css?v={{ gs_globals['git_commit'] }}" rel="stylesheet">
+
+
+<!-- Le HTML5 shim, for IE6-8 support of HTML5 elements -->
+<!--[if lt IE 9]>
+<script src="//html5shim.googlecode.com/svn/trunk/html5.js"></script>
+<![endif]-->
+
+</head>
+
+<body>
+  <nav class="navbar navbar-expand-xl navbar-light bg-light fixed-top border-bottom shadow-sm">
```

Comment:
The navbar is a bit thicker than the old one(50px), so we may have to tweak css in some places like the running note link, so that the headers don't go under the navbar.

Reply:
I'll try to remember this comment for the next PR when the font size change is in place.